tests/common/include/tests/common/tools/ray_gun.hpp

```diff
@@ -20,8 +33,8 @@ namespace detray {
  * @return a sorted vector of volume indices with the corresponding
  *         intersections of the portals that were encountered
  */
-template <typename detector_t, typename ray_t>
```

Comment:
@niermann999 
Let's just use only ray object for `ray_gun`. I don't see particular reason for generalizing the ray type